R/slice.R

```diff
@@ -304,8 +304,8 @@ vec_init <- function(x, n = 1L) {
 }
 
 # Used internally by `vec_rbind()`, but exported for testing
-vec_split_along <- function(x) {
-  .Call(vctrs_split_along, x)
+vec_split_along <- function(x, indices = NULL) {
```

Comment:
This doesn't seem like the right name now — maybe `vec_split_by()`? or `vec_split_with()`? Might be best to leave `vec_split_along()` and add a new function